src/monattack.cpp

```diff
@@ -3265,7 +2967,7 @@ bool mattack::nurse_operate( monster *z )
                 }
             }
         } else {
-            grab( z );
+            z->type->special_attacks.at( "grab" ).operator * ().call( *z );
```

Comment:
```suggestion
            z->type->special_attacks.at( "grab" )->call( *z );
```
same below and in the test unit